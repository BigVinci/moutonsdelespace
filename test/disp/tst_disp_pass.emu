#-------
#TEST_RETURN_CODE=PASS         
#TEST_COMMENT="Teste fonctionnement standard"
#-------

load elf/boucle.o
disp mem map
disp mem 0x1000 : 0xffffffff
disp reg all
disp reg $2
disp reg $LO $31 $14
disp mem 0x1000 : 0x1002 0x1004 : 0x1008