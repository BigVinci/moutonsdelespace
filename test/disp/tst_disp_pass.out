
------ Fichier ELF "elf/boucle.o" : sections lues lors du chargement ------
Virtual memory map (7 segments)
.text   	r-x	Vaddr: 0x00001000	Size: 24 bytes
.data   	rw-	Vaddr: 0x00002000	Size: 4 bytes
.bss    	rw-	Vaddr: 0x00003000	Size: 0 bytes
[heap]  	r--	Vaddr: 0x00005000	Size: 4286545920 bytes
[lib]   	r--	Vaddr: 0xff7fd000	Size: 8192 bytes
[stack] 	rw-	Vaddr: 0xff7ff000	Size: 8388608 bytes
[vsyscall]	r-x	Vaddr: 0xfffff000	Size: 4095 bytes
Symbol table has 7 syms:
Symbol            .text (0    bytes @ 0x00000000): section
Symbol            .data (0    bytes @ 0x00000000): section
Symbol             .bss (0    bytes @ 0x00000000): section
Symbol         .reginfo (0    bytes @ 0x00000000): section
Symbol             .pdr (0    bytes @ 0x00000000): section
Symbol           _start (0    bytes @ 0x00000000): object
Symbol                X (0    bytes @ 0x00000000): no type

Carte mémoire virtuelle (7 segments) : 
Virtual memory map (7 segments)
.text   	r-x	Vaddr: 0x00001000	Size: 24 bytes
.data   	rw-	Vaddr: 0x00002000	Size: 4 bytes
.bss    	rw-	Vaddr: 0x00003000	Size: 0 bytes
[heap]  	r--	Vaddr: 0x00005000	Size: 4286545920 bytes
[lib]   	r--	Vaddr: 0xff7fd000	Size: 8192 bytes
[stack] 	rw-	Vaddr: 0xff7ff000	Size: 8388608 bytes
[vsyscall]	r-x	Vaddr: 0xfffff000	Size: 4095 bytes
Affichage de la plage de données 1000 : ffffffff 

Affichage de la plage de données 1000 : 1018 incluse dans le segment .text
00001000  20  09  00  0c  00  00  50  20  21  4a  00  01  15  49  ff  fe  
00001010  00  00  00  00  00  00  50  20  

Affichage de la plage de données 2000 : 2004 incluse dans le segment .data
00002000  00  00  00  04  
 $0 : 0  $1 : 0  $2 : 0  $3 : 0 
 $4 : 0  $5 : 0  $6 : 0  $7 : 0 
 $8 : 0  $9 : 0  $10 : 0  $11 : 0 
 $12 : 0  $13 : 0  $14 : 0  $15 : 0 
 $16 : 0  $17 : 0  $18 : 0  $19 : 0 
 $20 : 0  $21 : 0  $22 : 0  $23 : 0 
 $24 : 0  $25 : 0  $26 : 0  $27 : 0 
 $28 : 0  $29 : 0  $30 : 0  $31 : 0 
 $32 : 0  $33 : 0  $34 : 0 
 $2 : 0  
 $34 : 0  
 $31 : 0  
 $14 : 0  
Affichage de la plage de données 1000 : 1002 

Affichage de la plage de données 1000 : 1002 incluse dans le segment .text
00001000  20  09  
Affichage de la plage de données 1004 : 1008 

Affichage de la plage de données 1004 : 1008 incluse dans le segment .text
00001004  00  00  50  20  
